pom.xml

```diff
@@ -98,8 +98,8 @@
     <javax.validation.api>2.0.1.Final</javax.validation.api>
     <asm.version>9.2</asm.version>
     <excludedGroups />
-    <memoryMb>1800</memoryMb>
-    <directMemoryMb>3000</directMemoryMb>
+    <memoryMb>2000</memoryMb>
+    <directMemoryMb>2500</directMemoryMb>
```

Comment:
Not sure 2500 is enough. Drill can use more. If you will run several times tests with this configs sucesfully - it is fine. Otherwise try to decrease slightly - 100-200Mb

Reply:
@vdiravka do you mean _increase_ slightly?